serial_protocol.md

```diff
@@ -29,9 +33,11 @@ Only needs to be called when the chiplet is in a new environment and needs to co
 	<< +CONNECTED
 	>> NETWORK home-guest
 	<< -UNABLE_TO_CONNECT
+	
+#Database Protocol
 
-##BEGIN
-Must be called after creating a Serial connection, it can take either just a host for accessing public variables or you may also provide a secret for accessing protected variables in the database.
+##BEGIN_DB
```

Comment:
Let's think about it some more. I am not going to block on it. We will need to change the database implementation to match so it could be a followup PR.